libkbfs/coalescing_context.go

```diff
@@ -1,50 +1,57 @@
 package libkbfs
 
 import (
-	"golang.org/x/net/context"
 	"reflect"
+	"time"
+
+	"golang.org/x/net/context"
 )
 
 // CoalescingContext allows many contexts to be treated as one.  It waits on
 // all its contexts' Context.Done() channels, and when all of them have
 // returned, this CoalescingContext is canceled. At any point, a context can be
 // added to the list, and will subsequently also be part of the wait condition.
-// TODO: add timeout channel in case there is a goroutine leak
+// TODO: add timeout channel in case there is a goroutine leak.
 type CoalescingContext struct {
 	context.Context
-	closeCh  chan struct{}
 	doneCh   chan struct{}
 	mutateCh chan context.Context
 	selects  []reflect.SelectCase
 }
 
+const (
+	mutateChanSelectIndex       int = 0
+	closeChanSelectIndex        int = 1
+	numExplicitlyHandledSelects int = 2
```

Comment:
If you used `iota` you wouldn't need to explicitly update this count (as long as it was always last).  But I can understand if you want to explicitly manage the indices in case the `iota` behavior even starts from something other than 0.